doc/additional_commands/index.rst

```diff
@@ -0,0 +1,194 @@
+
+Additional Commands
+===================
+
+Pylint ships with some additional tools that can be executed from the command line once Pylint itself is installed.
+
+
+Pyreverse
+---------
+
+``pyreverse`` analyzes your source code and generates package and class diagrams.
+It supports output to ``.dot``/``.gv``, ``.vcg`` and ``.puml``/``.plantuml`` (PlantUML) file formats.
+If Graphviz (or the ``dot`` command) is installed, all `output formats supported by Graphviz <https://graphviz.org/docs/outputs/>`_
+can be used as well. In this case, ``pyreverse`` first generates a temporary ``.gv`` file, which is then
+fed to Graphviz to generate the final image.
+
+Running Pyreverse
+'''''''''''''''''
+
+To run ``pyreverse``, use::
+
+  pyreverse [options] <packages>
+
+<packages> can also be a single Python module.
+The following options are availabe:
+
+.. option:: -h, --help
```

Comment:
I think we should advertise the help option only and make it exhaustive. Or we could generate the documentation using the code so there is something in the online doc ?

Reply:
I see your point. Describing all the options by hand might lead to doc and implementation becoming out of sync.

We could create a new script beside the existing ones at ``doc/exts`` and generate the documentation similar as in ``pylint_extensions.py``, but that seems a bit overkill as in contrast to the optional checkers ``pyreverse``'s options will remain relatively static. Maybe we could in some future MR implement something similar to ``sphinx-argparse``, but until then I will go with just advertising the ``-h`` option.